### PR TITLE
UI: Add warning if starting the output fails

### PR DESCRIPTION
**Pull request by derrod**
This is intended to address https://obsproject.com/mantis/view.php?id=789

Currenlty obs fails silently which could go unnoticed for users who are using hotkeys as well as confuse users who are not trained to read their logs when issues occur. With this users will get audible and visual feedback that something went wrong.

Suggestions for better, perhaps more clearer, error messages are welcome.

### CODE REVIEW

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

**Comment by SuslikV** [comment]
From the one side, I can think only of "notification message" in that case (if tray icon enabled).
From other side, I heard, that users are common with two displays these days, thus the status bar and buttons of the OBS Studio is always visible...

Also, popup on driver reboot (green splash) is possible (it is still happens to users), connection lost, reconnecting, reconnection success popup, capturing errors popup and many other popups. This things are destructive to streaming/recording.

In short - I don't like the idea of popups.

**Comment by derrod** [comment]
It is the kind of error where you want a clear and absolutely unambiguous notification that something went wrong. A popup notification is the proper way to do that.

This error will also only ever appear if streaming/recording fails to start once you click the button/hotkey. Never while streaming or recording is already active.

**Comment by SuslikV** [comment]
Recording and streaming can run independently and may start at any time (read it as "when other one is already active").

Never thought that you can start recording and never check is it active or not (it is like light signal that cam is active). That's why I don't like the popups with this kind of messages. Is this point of view clear?
